ENIGMAsystem/SHELL/Universal_System/Extensions/FileDropper/FileDropper.cpp

```diff
@@ -96,9 +96,18 @@ LRESULT CALLBACK SetHook(int nCode, WPARAM wParam, LPARAM lParam) {
 }
 
 HHOOK InstallHook() {
+  // Windows Vista
+  #if (_WIN32_WINNT == _WIN32_WINNT_VISTA)
```

Comment:
Are you sure this is working locally though? I don't know if `_WIN32_WINNT` is actually being defined by anything.
https://docs.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt?view=vs-2019

Reply:
It works with microsoft's compiler with my GM extension which effectively does the same thing, and I didn't anticipate MinGW being this big of a turd that it wouldn't support it.